anaconda_build/README.md

```diff
@@ -74,57 +74,49 @@ local development. E.g. on uframe-3-test, mi-instrument is installed under the r
 $ conda install mi-instrument
 ```
 
-## The stream/ingest engines and environment
+## The stream/ingest recipes, packages and environment
 
-The recipes are located in the <i>enginemeta</i>, <i>ingestEngine</i> and <i>streamEngine</i> folders. The enginemeta recipe declares the dependencies used while installing the packages built from the other recipes (ingestEngine and streamEngine). Modify it only as those dependencies change. The other recipes are based on the whole codebases via clones of the following Git repositories.
+These recipes are located in the <i>enginemeta</i>, <i>ingest_engine</i> and <i>stream_engine</i> folders. The enginemeta recipe declares the dependencies required for use in an environment that can be used to run the code obtained from the ingest_engine and stream_engine recipes. Modify it only as those dependencies change.
```

Comment:
any reason for not using the markdown syntax for italics?  ie \*ingest_engine\*

Reply:
Ignorance. I modified every place that I was making verbiage appear in italics to use the markdown syntax